pkg/pmem-device-manager/pmd-manager.go

```diff
@@ -1,5 +1,36 @@
 package pmdmanager
 
+import (
+	"errors"
+	"os"
+)
+
+var (
+	// ErrUnknown an unknown error occurred
+	ErrUnknown = errors.New("unknown error")
+
+	// ErrInvalid invalid argument passed
+	ErrInvalid = os.ErrInvalid
+
+	// ErrPermission no permission to complete the task
+	ErrPermission = os.ErrPermission
+
+	// ErrDeviceExists device with given id already exists
+	ErrDeviceExists = errors.New("device exists")
+
+	// ErrDeviceNotFound device does not exists
+	ErrDeviceNotFound = errors.New("device not found")
+
+	// ErrDeviceInUse device is in use
+	ErrDeviceInUse = errors.New("device is in use")
```

Comment:
"device in use"? Shorter, consistent with "device not found" and "device not ready" (i.e. leave out verb which isn't needed).

Reply:
Done.